packages/common/src/helpers.ts

```diff
@@ -32,3 +33,20 @@ export function useFilesToBeExempted(): string[] {
     }
     return [];
 }
+
+export function useCustomRules(): string[] {
+    const filePath = process.env.SA11Y_CUSTOM_RULES ?? '';
+    if (filePath !== '') {
+        try {
+            // Read the file synchronously
+            const data = fs.readFileSync(filePath, 'utf-8');
+            const jsonData = JSON.parse(data) as { rules: string[] };
+            // Access the rules array
+            const rules = jsonData.rules;
```

Comment:
we can minimize this 2 lines of code to one
`const { rules } = JSON.parse(data) as { rules: string[] };`